src/languages/en.ts

```diff
@@ -3451,13 +3451,13 @@ const translations = {
                 expense: 'Expense',
                 reimbursableExpenses: 'Export reimbursable expenses as',
                 nonReimbursableExpenses: 'Export non-reimbursable expenses as',
+                defaultPaymentAccount: 'NSQS default',
+                paymentAccount: 'Payment account',
+                paymentAccountDescription:
+                    'Choose the account that transactions will be approved against in NSQS. If you’re syncing reimbursed reports, this is also the account that bill payments will be created against.',
```

Comment:
@zsgreenwald Asking about copy here: https://expensify.slack.com/archives/C080703DZU6/p1739468991016639?thread_ts=1739403849.983999&cid=C080703DZU6

Reply:
Okay we should change this:
```suggestion
                    'Choose the account that will be used as the payment account for transactions NSQS.',
```

Same for the Spanish translation.

Reply:
Updated! As for the Spanish translation I have added it to the NSQS list to be reviewed as well https://expensify.slack.com/archives/C01GTK53T8Q/p1739580531000379?thread_ts=1738148924.210689&cid=C01GTK53T8Q